tests/src/fixMissingDoocs_eq_res.h

```diff
@@ -0,0 +1,17 @@
+// SPDX-FileCopyrightText: Deutsches Elektronen-Synchrotron DESY, MSK, ChimeraTK Project <chimeratk-support@desy.de>
+// SPDX-License-Identifier: LGPL-3.0-or-later
+#pragma once
+
+#warning Using copied code of doocs::comp_code due to bug in DOOCS 3.12.0. Switch to fixed API once available.
```

Comment:
Please don't add a warning here, instead write a redmine ticket so we do not forget to change this. The warning might hide other warnings, because things will be yellow all the time.

Reply:
done